src/moduler/aktivitet/visning/hjelpekomponenter/detaljvisning-lenke.tsx

```diff
@@ -10,8 +10,8 @@ interface Props {
 }
 
 export default function DetaljvisningLenke(props: Props) {
-    const lenke = props.lenke;
-    if (!lenke || !lenke.trim()) {
+    const lenke = props.lenke ? props.lenke.trim() : undefined;
```

Comment:
Hva med å bare gjøre sånn her?
```suggestion
    const lenke = props.lenke?.trim();
```